libwhisper/WhisperPeer.cpp

```diff
@@ -32,7 +31,11 @@ using namespace dev::shh;
 WhisperPeer::WhisperPeer(Session* _s, HostCapabilityFace* _h, unsigned _i, CapDesc const&): Capability(_s, _h, _i)
 {
 	RLPStream s;
-	sealAndSend(prep(s, StatusPacket, 1) << version());
+	//sealAndSend(prep(s, StatusPacket, 1) << version());
```

Comment:
It's safe to remove this. However, please also bump the protocol version for whisper capability. This will cause the capability to be disabled for older client versions; accordingly, we won't need the commented out code.